src/main/scala/nodes/images/CenterCornerPatcher.scala

```diff
@@ -0,0 +1,48 @@
+package nodes.images
+
+import org.apache.spark.rdd.RDD
+
+import utils.{ImageUtils, Image}
+import pipelines.FunctionNode
+
+/**
+  * Extract four corner patches and the center patch of the specified size.
+  * If flips is set to true, then horizontal flips of all 5 patches is also
+  * returned
+  *
+  * @param patchSizeX size of patch along xDim
+  * @param patchSizeY size of patch along yDim
+  * @param horizontalFlips if horizontal flips of patches should also be returned
+  * @return patches of size patchSizeX x patchSizeY
+  */
+case class CenterCornerPatcher(
```

Comment:
Can this be accomplished with a correctly configured `Windower`,  `HorizontalFlips`, and a `Gather` - not saying we wouldn't also want a node called this that is the composition of these things - just trying to avoid having a node that feels pretty special purpose.